src/sentry/web/client_config.py

```diff
@@ -158,18 +158,18 @@ def _resolve_last_org(
 
 
 class _ClientConfig:
+    user: User | AnonymousUser | None = None
+    session: SessionBase | None = None
+
```

Comment:
typically these don't need to be annotated on self -- if there does need to be more specific annotations they should be done in `__init__` during the assignments to `self` (that is, I think the original code was correct but perhaps needs an annotation on the first `self.user =` and `self.session =` lines

Reply:
The annotation is kind of required since type infer doesn't work as expected in this case.

Ex:

```
Mypy: Item "AnonymousUser" of "Any | AnonymousUser" has no attribute "email"  [union-attr]
```

Reply:
ah yeah but these should still go on `self` rather than outside of `__init__` on the class